### DISCUSSION TITLE
AuthorizationError: ('invalid_client', 'Client authentication failed', 401)

### DISCUSSION THREAD

**Discussion by dhidary** [saleweaver/python-amazon-sp-api#184] [Q&A] [answered]
Hey Getting this error, is something wrong with my code or is it just credentials?
Code:
`from sp_api.api import Orders
from sp_api.api import Reports
from sp_api.api import Feeds
from sp_api.base.reportTypes import ReportType
from datetime import datetime, timedelta

credentials=dict(
        refresh_token='<...>',
        lwa_app_id='<...>',
        lwa_client_secret='<...>',
        aws_secret_key='<...>',
        aws_access_key='<...>',
        role_arn='<...>',
    )

#idk why I put this here or if I need it pretty sure I don't
marketplace_id = ['ATVPDKIKX0DER']

print(credentials)
res  = Orders(credentials=credentials).get_orders(CreatedAfter=(datetime.utcnow() - timedelta(days=7)).isoformat())
print(res.payload)  # json data
`
Any help would be great, thanks!

**Comment by miogarun** [comment]
Try using your lwa_client_id in lwa_app_id

**Comment by dhidary** [comment, marked as answer]
Fixed the problem you have to remove the <> brackets
